lib/easy-stackup.js

```diff
@@ -0,0 +1,76 @@
+// simpler API main function
+'use strict'
+var gerberToSvg = require('gerber-to-svg')
+var shortId = require('shortid')
+var whatsThatGerber = require('whats-that-gerber')
+
+var pcbStackup = require('./index')
+
+module.exports = function(layers, optionsOrCallback, done) {
```

Comment:
This whole optional parameters in js boilerplate is annoying enough as is, so I try to keep it as small as possible:

``` js
module.exports = function(layers, options, done) {
  if (typeof options === 'function') {
    done = options
    options = {}
  }

  // ...
```